examples/credential_offer_multiple_credentials.json

```diff
@@ -12,8 +12,10 @@
          "issuer_state": "eyJhbGciOiJSU0Et...FYUaBy"
       },
       "urn:ietf:params:oauth:grant-type:pre-authorized_code": {
-         "pre-authorized_code": "adhjhdjajkdkhjhdj",
-         "user_pin_required": true
+         "pre-authorized_code": "oaKazRN8I0IbtZ0C7JuMn5",
```

Comment:
Did you consider grouping these properties in an object?
~~~
"user_pin": {
    "required": true,
    "length": 4,
    "description": "Please provide the PIN which was sent via e-mail"
}
~~~
It may result in a small string and logically seems to make sense. We could even remove the `"required": true` which would be implicit if the object is present.
The downside is that it would be a breaking change.

Reply:
While I agree that in most cases the locale can be determined from the context and be implicit, I'm afraid that we are making it impossible for an implementation to provide offers with more than one locale, in the eventual rare cases where it is needed.
Also, this seems to be the only place in this set of specs where we are providing localized user-facing strings without locale information. 
Note sure if saving some bytes justifies this exception.

Reply:
+1 to this structure. 
```
"user_pin": {
    "required": true,
    "length": 4,
    "description": "Please provide the PIN which was sent via e-mail"
}
```

Reply:
@pmhsfelix @Sakurann I applied the suggestion moving the 3 claims into one object.

Reply:
@pmhsfelix The discussion on i18n already happened in #10 and in DCP workshop at iiw. I'm unsure if this is now the best place to open this discussion again. Otherwise please make a concrete proposal, because this PR was the consensus

Reply:
If this is blocking for you please leave this in a dedicated review

Reply:
Agree with the current design. I guess we can add a new internationalized field if that is needed in the future.